webpack/main.ts

```diff
@@ -7,16 +7,20 @@ import path from "path";
 import type webpack from "webpack";
 import ForkTsCheckerPlugin from "fork-ts-checker-webpack-plugin";
 import nodeExternals from "webpack-node-externals";
-import * as vars from "./src/common/vars";
-import getTSLoader from "./src/common/getTSLoader";
+import getTypeScriptLoader from "./get-typescript-loader";
 import CircularDependencyPlugin from "circular-dependency-plugin";
-import { iconsAndImagesWebpackRules } from "./webpack.renderer";
+import { iconsAndImagesWebpackRules } from "./renderer";
+import type { WebpackPluginInstance } from "webpack";
```

Comment:
We have exported types from `webpack` above btw. 
Moreover I would better to remove this at all and mock every package with wrong or missing types in `mocks.d.ts`

Reply:
If https://github.com/import-js/eslint-plugin-import/pull/2419 gets merged then we can switch to using that instead so this isn't really that important.

I also don't know if it is worth it making the types in `mocks.d.ts`. Here it is explicit that something is strange.